### ISSUE TITLE
Windows 10

### ISSUE THREAD

**Issue by kursion** [rhiokim/haroopad#561]
I would love to give a try on Haroopad. I'm on windows 10 and it doesn't seem to work. Anyone has the same problem ?

**Comment by christophe-coyard** [comment]
Yes, same problem. The process is started but the software is not displayed.

**Comment by kursion** [comment]
Exactly. Would be cool to fix it then :)

**Comment by walter-weinmann** [comment]
Same problem here.

**Comment by eyecatchup** [comment]
Subscribed. ;)

**Comment by k1ngf15h3r** [comment]
Installed today on two Windows 10 Build 1511 devices without any problems

**Comment by Philippe64** [comment]
Same problem w10 x64 pro build 1511 french version

**Comment by chkilel** [comment]
I installed 32 and 64 bit, both doesn't work.

**Comment by rhiokim** [comment]
I've made new pkg for next release. **it's release candidate version, not official**
- https://www.dropbox.com/sh/o0gsowztbuumoig/AABcDuphGXImM_i7Vb4hPVIla?dl=0

And it was work well in pure Window 7(64/32), 8(64/32) and 10(64/32)

**Comment by apaugam** [comment]
@rhiokim doesn't work for me :(

**Comment by GulBroz** [comment]
@rhiokim : RC 0.13.2 is working ok on Win7 64 (fr)
v0.13.1 did **not** work (process haroopad.exe was running, but no window)

**Comment by noraj** [comment]
:+1: 
0.13.1 Win10 64 bits : haroopad process running but software doesn't diplay.

**Comment by cibvia** [comment]
Seems to be similar problem as #489, I had the same problem on Windows 10 french edition.
I could fix it deleting the file Path\to\Haroo Studio\Haroopad\locales\fr.pak.
It then runs the English version of the program.

**Comment by gaelle8278** [comment]
Thanks cibvia! I deleted the fr.pak file and now haroopad works nice on my windows 10 french edition.

**Comment by tgloriod** [comment]
Thank you cibvia, gaelle8278.

Delete C:\Users...\AppData\Roaming\Haroo Studio\Haroopad\locales\fr.pak
